js/msp.js

```diff
@@ -1510,6 +1506,9 @@ MSP.crunch = function (code) {
             for (var i = 0; i<CONFIG.name.length; i++) {
```

Comment:
Not introducing a dependency on constants in the firmware is a valid concern (albeit I wouldn't expect this particular one to change too often).

Looking at the code in `serial_msp.c`, the maximum length for a MSP seems to be 249, so we should at least check for that limit.